TreeDB/documentservice/service_column_graph_buffer_test.go

```
@@ -0,0 +1,202 @@
+	cmd := exec.Command(os.Args[0], "-test.run=^TestServiceColumnGraphBufferedInsertCrashRecovery$")
+	cmd.Env = append(os.Environ(), helper+"="+dir)
+	if output, err := cmd.CombinedOutput(); err != nil {
+		t.Fatalf("crash helper: %v\n%s", err, output)
+	}
+	db, err := backenddb.Open(testBackendOptions(dir))
+	if err != nil {
+		t.Fatalf("reopen db: %v", err)
+	}
+	defer db.Close()
+	svc := New(collections.NewCollectionManager(db))
+	defer svc.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `svc.Close` is not checked

(errcheck)